### ISSUE TITLE
image not shown in ssg

### ISSUE THREAD

**Issue by AnsonCheng03** [qwikest/icons#14]
It's okay to display images on `yarn start` but not `npm run build && npm run build.server`

My code in layout.tsx is
```
import { component$, Slot } from "@builder.io/qwik";

import Header from "~/components/header/header";
import FloatingContact from "~/components/floatingContact/floatingContact";
import Footer from "~/components/footer/footer";

export default component$(() => {
  return (
    <>
      <Header />
      <main>
        <Slot />
      </main>
      <FloatingContact />
      <Footer />
    </>
  );
});
```

When I enter the following code in `<Slot />`, it has no problem.
```
<Link href="https://wa.me/xxxxxx" target="_blank">
       <SiWhatsapp />
</Link>
``` 
However, the image is disappear in `<FloatingContact />` and `<Footer />` with the same code only in SSG.

**Comment by AnsonCheng03** [comment]
possibility related to #3

**Comment by quentin-salerne** [comment]
I think it's the same issue than #3

**Comment by hsnfirdaus** [comment]
Same here, it doesn't give any error on build. When page load, the icon blank (only comment in html dom). But after trigger rerender (signal change), icon showed.

**Comment by NiklasPor** [comment]
Duplicate of #3

**Comment by NiklasPor** [comment]
This should be fixed in `>= @qwikest/icons@0.0.9`. If not please comment here or write a new issue ❤️

**Comment by Kocal** [comment]
Thanks :)
